server-environment.md

```diff
@@ -26,52 +26,54 @@ PHP is a programming language on which WordPress code is based. This language ru
 
 WordPress supports many versions of PHP, some even obsolete, we recommend running PHP version 7.4 or higher:
 
-* [PHP 8.0](https://www.php.net/ChangeLog-8.php#PHP_8_0)
 * [PHP 7.4](https://www.php.net/ChangeLog-7.php#PHP_7_4)
+* [PHP 8.0](https://www.php.net/ChangeLog-8.php#PHP_8_0)*
+* [PHP 8.1](https://www.php.net/ChangeLog-8.php#PHP_8_1)*
 
-Versions prior to PHP 7.4 are not recommended because it doesn't have support of any kind. Security support for PHP 7.3 ends in Dec 6th 2021 which means that even if there are security problems with it, new version won't be released. Keeping your PHP to the latest stable version is important for WordPress speed and security.
+_IMPORTANT: WordPress 6.0 has beta support for PHP 8.0 and PHP 8.1. If used some of these versions may get some Warnings._
+
+Versions prior to PHP 7.4 are not recommended because it doesn't have support of any kind. Security support for PHP 7.3 ended in 2021-12-06 which means that even if there are security problems with it, new version won't be released. Keeping your PHP to the latest stable version is important for WordPress speed and security.
```

Comment:
Versions prior to PHP 7.4 do get (security) support from Linux distributions per their support policies. Even security patches are backported if needed. But support from PHP Group has ceased.

Reply:
Updated